src/variables.py

```diff
@@ -12,6 +12,8 @@
 CONSENSUS_CLIENT_URI: Final = os.getenv('CONSENSUS_CLIENT_URI', '').split(',')
 KEYS_API_URI: Final = os.getenv('KEYS_API_URI', '').split(',')
 
+TEST_CONSENSUS_CLIENT_URI: Final = os.getenv('TEST_CONSENSUS_CLIENT_URI', '').split(',')
```

Comment:
Let's move this variable to the tests somewhere, and add to docs/testing.md details about this env.